conf/defaults.ini

```diff
@@ -498,6 +498,10 @@ whitelist =
 headers =
 enable_login_token = false
 
+#################################### Auth JWT ##########################
+[auth.jwt]
+enabled = false
```

Comment:
All possible JWT settings should be included here for consistency, see `[auth.ldap]` or `[auth.proxy]` for examples. Also see comment in sample.ini